raw and with client apps (#2)
golemfactory/wasm-store: README.md

programs that are known to successfully work with
[Wasm sandbox](https://github.com/golemfactory/sp-wasm) in
[Golem](https://github.com/golemfactory/golem).

> The list includes applications directly located in this repo,

Reviewer: ```suggestion
The list includes applications located directly in this repo,
```

> as well as links that points to external.

Reviewer: ```suggestion
as well as links that point to external sources.
```

> The applications can be raw or augmented with a client.

Reviewer: ```suggestion
The applications can either be in a raw, Wasm format, or (preferably) they can be augmented with a GUI/CLI for the user's convenience.
```

> When using raw, a user connects directly with Golem. 

Reviewer: ```suggestion
Using raw Wasm binaries implies that the user has to be able to prepare the corresponding `task.json` and the required folder structure themselves, and be able to directly connect with their Golem client (e.g., via the use of the [Golem CLI](https://github.com/golemfactory/golem-client)). 
```

> That is more technical approach.

Reviewer: ```suggestion
Therefore, as such, this approach requires some technical knowledge of the Golem's internals.
```

See [here](https://docs.golem.network/#/About/Use-Cases?id=wasm)
to learn how to launch a Wasm task in Golem.

> The applications augmented with a client are more user friendly,

Reviewer: ```suggestion
The applications augmented with a GUI/CLI are naturally more user friendly,
```

> because it handles communication with Golem node,

Reviewer: ```suggestion
because they handle communication with Golem node,
```

> but there is no generic client. See g-flite for instance.

Reviewer: ```suggestion
Having said that, there currently is no generic way of preparing such a GUI/CLI. There are some examples however. See the [g-flite](https://github.com/golemfactory/g-flite) app for instance.
```

The list:

> * [g-flite](https://github.com/golemfactory/g-flite) - text-to-speech

Reviewer: @lukasz-glen How about we create 2 lists then: 1) with raw WASM binaries, and 2) with GUI/CLI client apps such as `g-flite`?

Author: sounds good

## Contributing
We welcome contributions in the form of links to precompiled Wasm binaries of
other programs. If you would like to submit such a link, do not hesitate to open a new PR.
Your repo should contain README file and license. 

> If it is raw application, it should follow the below guidelines.

Reviewer: ```suggestion
If it is a raw Wasm binary, it should follow the guidelines below.
```

> Approach to applications augemented with clients is more liberal.

Reviewer: ```suggestion
For apps augmented with GUI/CLI, the requirements are more relaxed and not set in stone, with the only must-have: good user experience :-)
```

## Directories structure

> When contributing an application in raw flavor, please make sure the repo the link will be pointing to adheres to the

Reviewer: ```suggestion
When contributing an application in a raw Wasm format, please make sure that the submitted link adheres to the
```
